src/client/cli/cmd/restart.cpp

```diff
@@ -46,7 +46,16 @@ mp::ReturnCode cmd::Restart::run(mp::ArgParser* parser)
         return standard_failure_handler_for(name(), cerr, status);
     };
 
-    spinner.start(instance_action_message_for(request.instance_names(), "Restarting "));
+    auto streaming_callback = [this, &spinner](mp::RestartReply& reply) {
+        if (!reply.log_line().empty())
+        {
+            spinner.print(cerr, reply.log_line());
+        }
+
+        spinner.stop();
+        spinner.start(reply.reply_message());
+    };
```

Comment:
i see there are some similar lines in `start.cpp` and they are covered. could these be covered in the same way?